clients/render/index.js

```diff
@@ -63,11 +63,15 @@ domReady(function () {
 
 		jQuery(document).on('cf.ajax.request', (event, obj) => {
 
-			//do not run if component is empty
-			if( theComponent.props.fieldsToControl.length <= 0 ){
+			//do not run if component don't have fieldsToControl
+			if( ! theComponent ){
 				return;
+			} else if( typeof theComponent === "object" ){
+				if(theComponent.props.fieldsToControl.length <= 0){
```

Comment:
@New0 If `theComponent` is a `null` then your conditional on 69 will pass. Seams silly but try it https://jsfiddle.net/ouejgzrm/

Why not `'object' === type of theComponent && 'function' === typeof theComponent.getAllFieldValues()` ?